templates/CRM/Contribute/Page/UserDashboard.tpl

```diff
@@ -65,7 +65,7 @@
                                 <a class="button no-popup nowrap"
                                    href="{crmURL p='civicrm/contribute/invoice' q=$urlParams}">
                                     <i class="crm-i fa-print"></i>
-                                    {if $row.contribution_status != 'Refunded' && $row.contribution_status != 'Cancelled' }
+                                    {if $row.contribution_status_id != 7 && $row.contribution_status_id != 3 }
```

Comment:
I think the normal syntax would be `neq` instead of `!=`

Reply:
Hmm - it always used to be that smarty writers used that format - then we discovered 'normal php' format worked & now it's pretty mixed. My preference is to migrate to this format on the basis it's more familiar to php / js writers and the smarty format is not really familiar to anyone